WickedEngine/wiGraphicsDevice_Vulkan.cpp

```diff
@@ -2004,6 +1996,20 @@ using namespace vulkan_internal;
 					}
 					break;
 
+					case VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER:
```

Comment:
why not default?

Reply:
from the preview in the discussion tab I totally mistook the code with another component. default is definitely better.